contributors/guide/contributor-cheatsheet/README-hi.md

```diff
@@ -1,35 +1,36 @@
+<!-- omit in toc -->
 # कुबेरनेट्स योगदानकर्ता चीट शीट
 
 कुबेरनेट्स परियोजना में उपयोग किए जाने वाले कुबेरनेट्स,टिप्स, ट्रिक्स और सामान्य सर्वोत्तम प्रथाओं में योगदान करते समय सामान्य संसाधनों की एक सूची। यह एक "TL;DR" या आपके GitHub योगदान अनुभव को बेहतर बनाने के लिए उपयोगी जानकारी का त्वरित संदर्भ है।
 
 **विषय - सूची**
 
-- [सहायक संसाधन](#Helpful-Resources)
-  - [आरंभ करना](#Getting-Started)
-  - [SIGS और अन्य समूह](#SIGS-and-Other-Groups)
-  - [समुदाय](#Community)
-  - [महत्वपूर्ण ईमेल उपनाम](#Important-Email-Aliases)
-  - [वर्कफ़्लो](#Workflow)
-  - [परीक्षण](#Tests)
-- [GitHub पर प्रभावी रूप से संवाद](#Communicating-Effectively-on-GitHub)
-  - [अन्य उपयोगी लिंक्स](#Other-Useful-Links)
-  - [कैसे एक-दूसरे के लिए उत्कृष्ट](#How-to-be-Excellent-to-Each-Other)
-    - [अच्छे / बुरे संचार के उदाहरण](#Examples-of-GoodBad-Communication)
-- [कंट्रीब्यूशन सबमिट करना](#Submitting-a-Contribution)
-  - [CLA पर हस्ताक्षर करना](#signing-the-CLA)
-  - [मुद्दों को खोलना और जवाब देना](#Opening-and-Responding-to-Issues)
-    - [एक मुद्दा बनाना](#Creating-an-Issue)
-    - [एक मुद्दे पर प्रतिक्रिया](#Responding-to-an-Issue)
-  - [एक Pull Request खोलना](#Opening-a-pull-Request)
-    - [एक Pull Request बनाना](#Create-a-Pull-Request)
-    - [उदाहरण PR विवरण](#Example-PR-Description)
-    - [एक Pull Request समस्या निवारण](#Troubleshooting-a-Pull-Request)
-  - [लेबल](#Labels)
-- [स्थानीय स्तर पर काम करना](#Working-Locally)
-  - [शाखा रणनीति](#Branch-Strategy)
-    - [Upstream जोड़ना](#Adding-Upstream)
-    - [अपने Fork को सिंक में रखना](#Keeping-Your-Fork-in-Sync)
-  - [स्क्वाशिंग कमिट्स](#Squashing-Commits)
+- [सहायक संसाधन](#सहायक-संसाधन)
+  - [आरंभ करना](#आरंभ-करना)
+  - [SIGS और अन्य समूह](#SIGS-और-अन्य-समूह)
+  - [समुदाय](#समुदाय)
+  - [महत्वपूर्ण ईमेल उपनाम](#महत्वपूर्ण-ईमेल-उपनाम])
+  - [वर्कफ़्लो](#वर्कफ़्लो)
+  - [परीक्षण](#परीक्षण)
+- [GitHub पर प्रभावी रूप से संवाद](#GitHub-पर-प्रभावी-रूप-से-संवाद)
+  - [अन्य उपयोगी लिंक्स](#अन्य-उपयोगी-लिंक्स)
+  - [एक-दूसरे से उत्कृष्ट कैसे बनें](#एक-दूसरे-से-उत्कृष्ट-कैसे-बनें)
```

Comment:
@mrbobbytables I think this should `एक-दूसरे के लिए उत्कृष्ट कैसे बनें`

Reply:
suggestions applied to all 3 👍